src/tcp_address.cpp

```diff
@@ -622,33 +647,6 @@ int zmq::tcp_address_t::resolve (const char *name_, bool local_, bool ipv6_, boo
           addr_str [addr_str.size () - 1] == ']')
         addr_str = addr_str.substr (1, addr_str.size () - 2);
 
-    // Test the '%' to know if we have an interface name / zone_id in the address
```

Comment:
Well, IPv6 is disabled completely on all weird platforms. I don't think it is good to leave this code in. We can wait for @syenna's reply.